cob_omni_drive_controller/src/odom_plugin.cpp

```diff
@@ -44,7 +44,6 @@ class OdometryController: public GeomController<hardware_interface::JointStateIn
         return true;
```

Comment:
forgot to advertise odometry topic, add:

topic_pub_odometry_ = controller_nh.advertise<nav_msgs::Odometry>("odom", 1);